test/app.js

```diff
@@ -64,6 +64,10 @@ app.set('view engine', 'njk');
 app.use(router);
 app.use('/static', express.static(path.join(__dirname, 'resources')));
 
+router.get('/techne-icons.svg', (req, res) => {
+    res.sendFile(path.join(__dirname, '..', 'scss/icons/techne-icons.svg'));
```

Comment:
@saadmhybris The SVG does not need to be copied, this will find it when testing the icons page.